opensearch_single_kernel/managers/exclusions.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""OpenSearch Nodes Exclusions manager."""
+
+
+from opensearch_single_kernel.core.state import ClusterState
+from opensearch_single_kernel.managers.base import BaseManager
+from opensearch_single_kernel.workload.base import BaseWorkload
+
+
+class NodesExclusionsManager(BaseManager):
+    """OpenSearch Nodes Exclusions Manager."""
+
+    def __init__(self, state: ClusterState, workload: BaseWorkload):
+        self.name = "exclusions_manager"
+        self.state = state
+        self.workload = workload
```

Comment:
```suggestion
        super().__init__(state, workload)
        self.name = "exclusions_manager"
```
This version also works but I  suggest to initialize super instead of defining them here.